GameOfLife.java

```diff
@@ -84,23 +130,73 @@ public static int[][] evolve(int[][] board) {
 	// Assumes that i is at least 1 and at most the number of rows in the board - 1. 
 	// Assumes that j is at least 1 and at most the number of columns in the board - 1. 
 	// Uses the count(board,i,j) function to count the number of alive neighbors.
-	public static int cellValue(int[][] board, int i, int j) {
+	public static int cellValue(int[][] board, int i, int j) 
+	{
 		//// Replace the following statement with your code.
-		return 0;
+		int num = count (board,i,j);
+		int newValue = 0; 
+		
+		if (board[i][j] == 1 && num < 2)
+			newValue = 0;
+		else
+			if (board[i][j] == 1 && num == 3 || num == 2)
+				newValue = board[i][j];
+			else
+				if (board[i][j] == 1 && num >3)
+					newValue = 0;
+				else
+					if (board[i][j] == 0 && num == 3)
+					newValue = 1;
```

Comment:
Let's simplify it, since there's no need to update newValue when it should be 0.
In addition, always scope your code within `if\else` even when there is only one line of code inside.
```suggestion
		if (board[i][j] == 1 && (num == 3 || num == 2)) {
			newValue = 1;
		}
		else if (board[i][j] == 0 && (num == 3) ) {
			newValue = 1;
		}
```